context-core/pom.xml

```diff
@@ -64,30 +64,24 @@
 
         <!-- Jena dependencies -->
         <dependency>
-            <groupId>com.hp.hpl.jena</groupId>
-            <artifactId>arq</artifactId>
-            <version>2.8.7</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>log4j</groupId>
-                    <artifactId>log4j</artifactId>
-                </exclusion>
-            </exclusions>
+            <groupId>org.apache.jena</groupId>
+            <artifactId>jena-arq</artifactId>
```

Comment:
could be removed because it's pulled in by `jenabean` (and `jena-sdb`, `jena-tdb`) already

Reply:
need to be kept until we have replaced SDB by TDB. 
If we remove this entry in the POM, maven pulls the wrong version.